docker-compose.prod.yml

```
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  app:
+    image: ghcr.io/daritelska-platforma/frontend/web:deployment-changes
```

@miraclebg We only need to find a way to define the newly built version

```suggestion
    image: ghcr.io/daritelska-platforma/frontend/web:${DEPLOY_TAG}
```

And pass `DEPLOY_TAG` from the `TapTap21/docker-remote-deployment-action` 

Without such mechanism we the tag `deployment-changes` will act the same as tag `latest`

Any other ideas are welcome